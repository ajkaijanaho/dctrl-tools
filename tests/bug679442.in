Mergefield: 1
Duplicate-field: foo
