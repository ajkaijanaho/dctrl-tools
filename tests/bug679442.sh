#!/bin/sh

set -e

LC_ALL=C
export LC_ALL

$JOIN_DCTRL -j Mergefield bug679442.in bug679442.in
