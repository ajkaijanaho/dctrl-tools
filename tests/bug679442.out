Mergefield: 1
Duplicate-field: foo
Mergefield: 1
Duplicate-field: foo

